frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -37,12 +37,13 @@ local MODE = {
     book_time_to_read = 7,
     chapter_time_to_read = 8,
     frontlight = 9,
-    mem_usage = 10,
-    wifi_status = 11,
-    book_title = 12,
-    book_chapter = 13,
-    bookmark_count = 14,
-    chapter_progress = 15,
+    frontlight_warmth = 10,
+    mem_usage = 11,
+    wifi_status = 12,
+    book_title = 13,
+    book_chapter = 14,
+    bookmark_count = 15,
+    chapter_progress = 16,
```

Comment:
Please don't modify existing values, I vaguely remember discussing this in the past, we allowed it, and it came back to bite us in the ass later ;).